src/workloads/query/multiplanner/NoResults.yml

```diff
@@ -6,10 +6,6 @@ Description: |
   are very selective (match 0% of the documents). With zero results, we do no hit the EOF optimization
   and all competing plans hit the works limit instead of document limit.
```

Comment:
Wait, reading this again I'm confused about the purpose of this workload. Won't every plan hit EOF immediately because the interval for the index scan is empty? If I'm reading things correctly, this note about not hitting the EOF optimization is wrong.

Reply:
Oops, you're right and we must have discussed this previously, because I had left the same comment on the google doc: https://docs.google.com/document/d/1hBJoIOwDMPZItTCvYTebfbACB62YwvhRCo7pP8Q_iy4/edit?disco=AAABM1bxvks

Reply:
If the goal were to hit max works, then a better test would be to have all the indexed predicates select 100% of documents, and have one residual predicate that selects 0%.  This is what `NoSuchField.yml` already does, so this case is covered.

I'll just update the comment here to describe what this file actually does.